packages/protocol/scripts/bash/contract-exclusion-regex.sh

```diff
@@ -18,10 +18,8 @@ if [ $VERSION_NUMBER -gt 8 ]
   CONTRACT_EXCLUSION_REGEX="$CONTRACT_EXCLUSION_REGEX|^Ownable|Initializable|BLS12_377Passthrough|BLS12_381Passthrough]UniswapV2ERC20"
 fi
 
-  # https://github.com/celo-org/celo-monorepo/issues/10435
-  # SortedOracles is currently not deployable
-  # after fixing that this should be modified to VERSION_NUMBER==10
```

Comment:
The comment here mentions to use `-eq 10`. Have we changed the approach such that 9 is indeed the correct number?

Reply:
Yes, since this is CR11, we need CR10 to already include SortedOracles when comparing. For CR9 we on the other hand need to ignore it (or we would have to introduce further tooling updates).